docs/docs.md

```diff
@@ -0,0 +1,269 @@
+Please contribute to this document if you think something is missing or could be better.
+
+# Contents
+
+- [Introduction](#introduction)
+- [The V to Android App Development Cycle](#the-v-to-android-app-development-cycle)
+- [Using `vab` from the command line](#using-vab-from-the-command-line)
+- [Using `vab` programmatically](#using-vab-programmatically)
+- [Examples](#examples)
+- [Compile V code to Android Compatible C code](#compile-v-code-to-android-compatible-c-code)
+- [Compile C code to Android shared library (.so)](#compile-c-code-to-android-shared-library-so)
+- [Find and Invoke NDK Compiler Manually](#find-and-invoke-ndk-compiler-manually)
+
+# Introduction
+
+Android is a, primarily, Java based Operating System - and to be able to develop apps for
+*the majority* of devices on the market (2022), the primary option is to target Google-based
+Android devices.
+
+To be able to develop for Google-based devices, Google provide the *Android SDK*, and
+for native developement (C/C++), they provide the *Android NDK*. Both are currently required
+to be able to develop Android apps in V.
+
+While the Android OS itself is open source - the SDK and NDK distributed by Google is ***not***.
+This fact makes everything a bit harder than it ought to be.
+
+So - for us to target the majority, we need the Android SDK and NDK that are
+both proprietary licensed in such a way that they prevent developers from
+modifying and re-distributing them - thus forcing developers to use Google's
+official (rather large) distributions of these developement kits.
+
+For historical reasons - the job of locating and using all the tools in the
+development kits is actually harder than it might appear at first sight
+since Google tend to move things around, deprecate tools on short, if any, notice
+and, in general, always be in a kind of "beta" mode when they release the software
+needed to target and build for the Google flavoured Android OS devices.
+
+To compensate for this they've developed a ~900MiB tool called "Android Studio"
+to aid developers in developing for their platform... Because of it's huge size,
+slow compile speeds and the difficulties involved in integrating a satisfying user
+experience for, at least, V developers - V has chosen to develop a tool that
+completely eliminate the need to use Android Studio in V development.
+
+With all this in mind `vab` was born to make life easier for V developers.
+You can install it by following the instructions [in the README.md](https://github.com/vlang/vab#install).
+
+Because of the situation described above and the design of the Android developer eco-system,
+`vab` is basically just a wrapper around the Android SDK and NDK. `vab` is able to both
+install, locate and orchestrate the tools found inside the distributed development kits.
+
+Welcome - and have a productive time using V and `vab`!
```

Comment:
That may be put on top.

Reply:
Good idea

Reply:
Done in [ebdda19](https://github.com/vlang/vab/pull/236/commits/ebdda192e01e196ad7edd2c3b0ef9d2d19965e88) along side, mostly, formatting changes